src/btree/bt_sync_obsolete.c

```diff
@@ -81,10 +80,10 @@ __sync_obsolete_inmem_evict_or_mark_dirty(WT_SESSION_IMPL *session, WT_REF *ref)
         if (addr.type == WT_ADDR_LEAF)
             ovfl_items = true;
         do_visibility_check = true;
-    } else
-        tag = "unexpected page state";
```

Comment:
Where did this code go?

Reply:
I see it is now initialised to this value in cf3298311bb2d497123dbd58ed5df83ba8ba74ae.